sw/device/silicon_creator/lib/crypto/rsa_3072/rsa_3072.s

```diff
@@ -438,6 +438,372 @@ modexp_var_3072_f4:
 
   ret
 
+/**
+ * Subtracts the modulus M from A.
+ *
+ * Flags: After this subroutine, the C flag is set to 1 if the subtraction
+ * underflowed.
+ *
+ * This routine runs in variable time.
+ * @param[in]  x16: dmem pointer to first limb of modulus M
+ * @param[in]  [w4:w15]: operand A
+ * @param[in]  w31: all-zero
+ * @param[out] [w16:w27]: result
+ *
+ * clobbered registers: x8 to x12, w2, w3, w16 to w27
+ * clobbered Flag Groups: FG0
+ */
+subtract_modulus_var:
+
+  /* prepare temporary registers */
```

Comment:
Sorry to be a stickler about this, but can you make sure comments start with an uppercase letter, end with a period, and use the `//` form? (Presumably the OTBN assembler is ok with those.)

Reply:
The assembler doesn't permit  `//`-form comments. But I'll change them to start with uppercase letters and end with periods.